pkg/persistence/account/loader/load.go

```diff
@@ -28,11 +29,11 @@ import (
 // Load loads account management resources from YAML configuration files
 // located within the specified root directory path. It parses the YAML files, extracts policies,
 // groups, and users data, and organizes them into a AMResources struct, which is then returned.
-func Load(fs afero.Fs, rootPath string) (*AMResources, error) {
-	resources := &AMResources{
-		Policies: make(map[string]Policy),
-		Groups:   make(map[string]Group),
-		Users:    make(map[string]User),
+func Load(fs afero.Fs, rootPath string) (*account.AMResources, error) {
```

Comment:
note that CI flags this method's cyclomatic complexity once again after this commit moved it. ignore that check